src/main/java/BadApple/task/Storage.java

```diff
@@ -0,0 +1,161 @@
+package BadApple.task;
+
+import BadApple.main.BadAppleException;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Manages the changes in tasks due to a user's commands.
+ * Commands are passed from the Parser into actual actions
+ * which this class then updates the program's task list.
+ * Currently, updateTask is called by Parser after each command.
+ */
+public class Storage {
+    /**
+     * After each request by the user, erases previous contents of the file
+     * and rebuilds the file based on task list
+     * @param file the file to write tasks to
+     * @throws IOException if file writing fails for any reason
+     */
+    public static void updateTasks(File file) throws IOException {
+        BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file));
+
+        // write to file
+        int index = 1;
+        for (Task t : TaskList.tasks) {
+            String task = index + " " + t.toString() + "\n";
+            bufferedWriter.write(task);
+            index++;
+        }
+
+        bufferedWriter.close();
+    }
+
+    /**
+     * Adds a task to the program's task list.
+     * Meant to display messages to signal a task being added
+     * except when first loading in a task
+     * @param task Task to add to the TaskList
+     */
+    public static void addTask(Task task) {
+        TaskList.tasks.add(task);
+        if (!Messenger.suppressMessages) {
+            System.out.println(Messenger.CustomMessages.randomMsg(task));
+            System.out.println(task);
+        }
+    }
+
+    /**
+     * Removes a task from the program's task list
+     * Usually called by the "delete" command
+     * @param i the index at which a task should be removed
+     */
+    public static void removeTask(int i) {
+        System.out.println("Exploooosion! now task " + TaskList.tasks.remove(i).brief() + " has been Kazuma-ed out of existence");
+        System.out.println("You now have " + TaskList.tasks.size() + " tasks in your list");
+    }
+
+    /**
+     * Loads in the local drive's save file of Tasks.
+     * given a File and its contents, add it into the Tracker's taskList
+     * @param file the file to read from
+     * @throws IOException when a file can't be read from
+     * @throws BadAppleException when the file contents are in the wrong format (i.e. non-command)
+     */
+    public static void parseTasks(File file) throws IOException, BadAppleException {
+        // check the file to see what tasks are already available.
```

Comment:
Would it be better to begin the comment with a capital letter and use proper sentence punctuation for improved readability? I noticed the same issue for some other comments too.